app/views/comments/index.erb

```diff
@@ -0,0 +1,12 @@
+<h5>Comments:</h5>
+    <% @suggestion.comments.each do |comment| %>
+        <%= comment.body %>
+        <% if comment.user_id == comment_user.id %>
```

Comment:
Heads up: local variable `comment_user` is not defined in this context. I think here you are meaning to do:
```erb
 <% if comment.user_id == current_user.id %>
```